stories/Section.stories.mdx

```diff
@@ -0,0 +1,95 @@
+import { Meta, Canvas, Story, ArgsTable } from '@storybook/addon-docs';
+import { linkTo } from '@storybook/addon-links';
+import { Section, Text } from '../src'; // components you used in the story
+
+import {
+    StandardSection,
+    RowSection,
+    StylingSection,
+    NavigationSection,
+} from './examples/Section';
+
+<Meta title="Interfacing/Section" component={Section} />
+
+# Section
+
+The Section component works like a flexbox grid container with a fixed `width` and `height`. It can be used to create a layout for your components.
```

Comment:
it works like `flexbox`. In this instance to mention the grid, because they work differently